hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -2460,6 +2460,14 @@
     </description>
 </property>
 
+<property>
+  <name>ipc.server.purge.interval</name>
+  <value>15</value>
+  <description>In Server, how often will Calls be cleaned up?
```

Comment:
nit: 
Can remove the question mark. Asking question in the description will not make sense. :-)
Please rephrase it just tell this configuration defines how often the calls will get cleaned up in the server.

Reply:
Thanks @ayushtkn for the comment.
I will make some changes later.